packages/flutter/lib/src/cupertino/app.dart

```diff
@@ -427,9 +429,27 @@ class CupertinoApp extends StatefulWidget {
 ///
 ///  * [ScrollBehavior], the default scrolling behavior extended by this class.
 class CupertinoScrollBehavior extends ScrollBehavior {
+  /// Creates a CupertinoScrollBehavior that uses [BouncingScrollPhysics] and
+  /// adds [CupertinoScrollbar]s on desktop platforms.
+  const CupertinoScrollBehavior({
+    Set<TargetPlatform>? scrollbarPlatforms,
+  }) : super(
+    scrollbarPlatforms: scrollbarPlatforms,
+    glowingPlatforms: const <TargetPlatform>{},
+  );
+
   @override
-  Widget buildViewportChrome(BuildContext context, Widget child, AxisDirection axisDirection) {
-    // Never build any overscroll glow indicators.
+  Widget buildViewportDecoration(
+    BuildContext context,
+    Widget child,
+    ScrollableDetails details,
+  ) {
+    if ((scrollbarPlatforms ?? ScrollBehavior.defaultScrollbarPlatforms).contains(getPlatform(context))) {
```

Comment:
Could `ScrollBehavior.defaultScrollbarPlatforms` just be the default value for `scrollbarPlatforms`?